src/Raven.Server/Documents/Handlers/QueriesHandler.cs

```diff
@@ -216,6 +216,21 @@ private async Task Explain(DocumentsOperationContext context, RequestTimeTracker
             }
         }
 
+        private async Task ServerSideQuery(DocumentsOperationContext context, RequestTimeTracker tracker, HttpMethod method)
+        {
+            var indexQuery = await GetIndexQuery(context, method, tracker);
+            using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBodyStream(), Database.DatabaseShutdown))
+            {
+                writer.WriteStartObject();
+                writer.WriteComma();
```

Comment:
This will generate invalid JSON. Can you remove the comma?

Reply:
tested this on fiddler didn't notice i accidentally copied the comma line